extensions/blocks/podcast-player/style.scss

```diff
@@ -176,28 +186,79 @@ $player-float-background: $light-gray-200;
 		&:focus {
 			color: $text-color-hover;
 		}
+
+		// Override editor styles
+		&:focus {
+			outline: none;
+			box-shadow: none;
+		}
```

Comment:
Focus styles are essential, as you need to have some indication of where the focus is. I think leaving them in and addressing the difference later is better than removing them.

Reply:
Agreed.